### DISCUSSION TITLE
220902 월터 일간회고

### DISCUSSION THREAD

**Discussion by sungman5** [codestates/BEB-05-Team-Vote#35] [일간회고]
### 오늘은 어떻게 프로젝트에 기여했나요?
- TypeScript 기초 수강을 완료했고, 심화 파트가 남아있습니다.
- NextJS 강의 50% 이상 수강했습니다.
- Jaden의 styled_component & And Design 리뷰를 보았습니다.

### 오늘의 프로젝트에서 힘든 점은 무엇인가요?
- 아직 타입스크립트의 타입 정의가 익숙하지 않아서 코드 작성이 느릿느릿합니다.
- 하지만 익숙해지기만 하면 너무 좋겠다는 생각이 계속 듭니다. 니꼴라스는 이제 자바스크립트로 돌아갈 수 없다던데
- 앤트디자인 라이브러리는 처음에는 굳이 이걸 써야 하나 싶었습니다만... 왜냐면 그냥 바닐라CSS로 작성하는게 더 빠를 것 같다는 생각이 들었기 때문입니다. 하지만 팀프로젝트이고, 저 혼자 CSS를 작성하다가 병목현상이 발생할 경우를 생각하니, CSS 라이브러리를 쓰는 것이 프로젝트 목적에 더 맞다는 생각이 들었습니다. 어서 익숙해지도록 하겠습니다.

### 내일은 프로젝트에 기여하기 위해 무엇을 해야 하나요?

- [ ] 와이어프레임을 바탕으로 사이트 UI 작업을 진행합니다.
- [ ] 타입스크립트와 NextJS 의 남은 강의를 수강 완료하고 반복학습합니다.
- [ ] 스타일드컴포넌트와 앤트디자인을 학습합니다.

**Comment by thyoondev** [comment]
ui 라이브러리를 이용하게되면 정말 생산성이 높아질거에요. 이번에 제대로 사용해보고 만약 월터에게 맞지 않는 것 같다면 월터에게 맞는 방법을 사용해보는 것도 좋을 것 같아요. 
사용하기 전까지는 확실히 좋은건지 아니면 필요없는 건지 잘 모를 수도 있으니까요 !
